build/ProjectReunion-Create-VSIX.yml

```diff
@@ -0,0 +1,96 @@
+parameters:
+  - name: "ReunionVersion"
+    type: string
+    default: ''
+  - name: "ReunionFoundationVersion"
+    type: string
+    default: ''
+  - name: "ReunionDWriteVersion"
+    type: string
+    default: ''
+  - name: "ReunionWinUIVersion"
+    type: string
+    default: ''
+  - name: "releaseBuild"
+    type: boolean
+    default: True
+
+jobs:
+
+- job: CreatingVsix
+  pool:
+    vmImage: 'windows-2019'
+
+  variables:
+    ${{if eq(parameters.releaseBuild, True) }}:
+      VSIXBuildArgs: '/p:"RestoreSources=https://api.nuget.org/v3/index.json;https://pkgs.dev.azure.com/microsoft/ProjectReunion/_packaging/Project.Reunion.Internal.ReleaseSigned/nuget/v3/index.json"'
+    ${{if eq(parameters.releaseBuild, False) }}:
+      VSIXBuildArgs: ''
+      
+  steps:
+  - task: NuGetAuthenticate@0
+    inputs:
+      nuGetServiceConnections: Internal-ReleaseSigned
+
+  # NuGetCommand@2 
+  - task: 333b11bd-d341-40d9-afcf-b32d5ce6f23b@2
+    displayName: 'NuGet restore'
+    inputs:
+      command: 'custom'
+      arguments: 'restore dev\VSIX\packages.config -PackagesDirectory $(UserProfile)\.nuget\packages'
+
+  - task: VSBuild@1
+    displayName: 'Restore ProjectReunion.Extension.sln'
+    inputs:
+      solution: dev\VSIX\ProjectReunion.Extension.sln
+      platform: 'Any CPU'
+      configuration: '$(buildConfiguration)'
+      msBuildArgs: '/t:restore /p:ReunionVersion="${{ parameters.ReunionVersion }}" /p:ReunionFoundationVersion="${{ parameters.ReunionFoundationVersion }}" /p:ReunionDWriteVersion="${{ parameters.ReunionDWriteVersion }}" /p:ReunionWinUIVersion="${{ parameters.ReunionWinUIVersion }}" ${{ variables.VSIXBuildArgs }}'
+
+  - task: VSBuild@1
+    displayName: 'Build ProjectReunion.Extension.sln'
+    inputs:
+      solution: dev\VSIX\ProjectReunion.Extension.sln
+      platform: 'Any CPU'
+      configuration: '$(buildConfiguration)'
+      msBuildArgs: '/p:VSToolsPath="$(UserProfile)\.nuget\packages\microsoft.vssdk.buildtools\16.9.1050\tools" /p:ReunionVersion="${{ parameters.ReunionVersion }}" /p:ReunionFoundationVersion="${{ parameters.ReunionFoundationVersion }}" /p:ReunionDWriteVersion="${{ parameters.ReunionDWriteVersion }}" /p:ReunionWinUIVersion="${{ parameters.ReunionWinUIVersion }}" ${{ variables.VSIXBuildArgs }}'
```

Comment:
VSToolsPath should not be supplied here - that's what's necessitating the packages.config above